core/server/lib/members/subscriptions/payment-processors/stripe/api.js

```diff
@@ -134,9 +150,10 @@ function createEnsurer(get, create, generateHashSeed) {
 function createApi(resource, validResult, getAttrs, generateHashSeed) {
     const get = createGetter(resource, validResult);
     const create = createCreator(resource, getAttrs);
+    const remove = createDeleter(resource, get, generateHashSeed);
     const ensure = createEnsurer(get, create, generateHashSeed);
 
     return {
-        get, create, ensure
+        get, create, remove, ensure
```

Comment:
@allouis Was opting to go for `del` instead of `remove` earlier, but to ensure consistency with rest of naming in `subscriptions` went with `remove`. Still partially inclined towards `del` as seems better in case of deletion, let me know what you think. 😄

Reply:
Personally I prefer `remove` because it's a full word, and we shouldn't use `delete` because it's a keyword :shrimp: 

Whatever we go with though should probably be either `remove = createRemover(` or `del = createDeleter(`